kitti360:
  data_root_dir: "/Datasets/KITTI-360/"
  data_2d_raw_dir: "data_2d_raw/"
  data_3d_raw_dir: "data_3d_raw/"
  data_calibration_dir: "calibration/"
  data_poses_dir: "data_poses/"
  data_poses_oxts_extract_dir: "data_poses_oxts_extract/"
  sequence_number: "0003"
  data_output_dir: "output/"